pkg/Microsoft.NETCore.Platforms/runtime.json

```diff
@@ -1686,6 +2097,140 @@
         "ubuntu-x86-corert"
       ]
     },
+    "ubuntu.17.04": {
+      "#import": [
+        "ubuntu"
+      ]
+    },
+    "ubuntu.17.04-arm": {
+      "#import": [
+        "ubuntu.17.04",
+        "ubuntu-arm"
+      ]
+    },
+    "ubuntu.17.04-arm-corert": {
+      "#import": [
+        "ubuntu.17.04-corert",
+        "ubuntu.17.04-arm",
+        "ubuntu.17.04",
+        "ubuntu-arm-corert"
+      ]
+    },
+    "ubuntu.17.04-arm64": {
+      "#import": [
+        "ubuntu.17.04",
+        "ubuntu-arm64"
+      ]
+    },
+    "ubuntu.17.04-arm64-corert": {
+      "#import": [
+        "ubuntu.17.04-corert",
+        "ubuntu.17.04-arm64",
+        "ubuntu.17.04",
+        "ubuntu-arm64-corert"
+      ]
+    },
+    "ubuntu.17.04-corert": {
+      "#import": [
+        "ubuntu.17.04",
+        "ubuntu-corert"
+      ]
+    },
+    "ubuntu.17.04-x64": {
+      "#import": [
+        "ubuntu.17.04",
+        "ubuntu-x64"
+      ]
+    },
+    "ubuntu.17.04-x64-corert": {
+      "#import": [
+        "ubuntu.17.04-corert",
+        "ubuntu.17.04-x64",
+        "ubuntu.17.04",
+        "ubuntu-x64-corert"
+      ]
+    },
+    "ubuntu.17.04-x86": {
+      "#import": [
+        "ubuntu.17.04",
+        "ubuntu-x86"
+      ]
+    },
+    "ubuntu.17.04-x86-corert": {
+      "#import": [
+        "ubuntu.17.04-corert",
+        "ubuntu.17.04-x86",
+        "ubuntu.17.04",
+        "ubuntu-x86-corert"
+      ]
+    },
+    "ubuntu.17.10": {
+      "#import": [
+        "ubuntu"
+      ]
+    },
+    "ubuntu.17.10-arm": {
+      "#import": [
+        "ubuntu.17.10",
+        "ubuntu-arm"
+      ]
+    },
+    "ubuntu.17.10-arm-corert": {
+      "#import": [
+        "ubuntu.17.10-corert",
+        "ubuntu.17.10-arm",
+        "ubuntu.17.10",
+        "ubuntu-arm-corert"
+      ]
+    },
+    "ubuntu.17.10-arm64": {
+      "#import": [
+        "ubuntu.17.10",
+        "ubuntu-arm64"
+      ]
+    },
+    "ubuntu.17.10-arm64-corert": {
+      "#import": [
+        "ubuntu.17.10-corert",
+        "ubuntu.17.10-arm64",
+        "ubuntu.17.10",
+        "ubuntu-arm64-corert"
+      ]
+    },
+    "ubuntu.17.10-corert": {
+      "#import": [
+        "ubuntu.17.10",
+        "ubuntu-corert"
+      ]
+    },
+    "ubuntu.17.10-x64": {
+      "#import": [
+        "ubuntu.17.10",
+        "ubuntu-x64"
+      ]
+    },
+    "ubuntu.17.10-x64-corert": {
+      "#import": [
+        "ubuntu.17.10-corert",
+        "ubuntu.17.10-x64",
+        "ubuntu.17.10",
+        "ubuntu-x64-corert"
+      ]
+    },
+    "ubuntu.17.10-x86": {
+      "#import": [
+        "ubuntu.17.10",
+        "ubuntu-x86"
+      ]
+    },
+    "ubuntu.17.10-x86-corert": {
```

Comment:
Could you also go ahead and add ubuntu.18.04 proactively?

Reply:
Similarly if you wouldn't mind, Fedora 28 and Mint 19 all come out around the same time and should be included in the 2.1 RID graph

Reply:
Sure. Do you think there are any other distro versions I should do that for?

Reply:
Thanks @bording.  With those additions you'll have added all of the supported distro versions through the ~May 2018 time frame.  

Near the end of 2018, we'll need to add the supported H2 2018  releases {Ubuntu18.10 / Mint 19 / SLES 15 / Fedora 29} to the master branch around that time; as well as stand up the appropriate build and test infra for those releases.